### DISCUSSION TITLE
[초조한 전망대] 온보딩 미션 참여 관련 질문입니다.

### DISCUSSION THREAD

**Discussion by yeoamlog** [Pseudo-Lab/Anxious-Lookout#14] [Q&A] [answered]
안녕하세요. 이번에 큰 맘 먹고 가짜연구소에 참여해서 활동해보고자 들어왔는데요.
디스코드로 접해서인지 어디서부터 어떻게 시작해야하는지 헤매고 있던 중 '초조한 전망대' 써클을 알게 되었습니다.
노션에서 참조하고, 깃헙으로 pr을 하라고 하셨는데요.
제가 깃을 전혀 모르던 상태라 조금씩 찾아보는 중인데 어떻게 하면 되는지 혹시 튜토리얼 가이드 같은 것이 있을까요!?

**Comment by HangryDev** [comment, marked as answer]
채팅으로만 답하고 댓글을 안 남겼네요!
필요하신 만큼 저희가 알려드려여! :)
